scalalib/worker/src/mill/scalalib/worker/ZincWorkerImpl.scala

```diff
@@ -492,28 +492,6 @@ class ZincWorkerImpl(
       .map(path => converter.toVirtualFile(path.toNIO))
       .toArray
 
-    // We want to listed to all compiled files
-    val compileProgress = reporter.map { reporter =>
```

Comment:
The issue is most likely here https://github.com/lampepfl/dotty/issues/13082

Wouldn't  CompileProgress be useful for reporting % of the compilation or does mill handle it differently?

Reply:
We are not reporting % of the compilation in Mill yet. I'm creating an issue so we don't forget about it.

Reply:
Thanks! Anyway, this needs the fix for Scala 3, but otherwise we can probably copy the logic from Bloop

Reply:
Issue created here: #2816